neuraxle/pipeline.py

```diff
@@ -119,15 +118,8 @@ def save(self, pipeline: 'Pipeline', data_container: DataContainer) -> 'Pipeline
         if not os.path.exists(pipeline_cache_folder):
             os.makedirs(pipeline_cache_folder)
 
-        with open(os.path.join(pipeline_cache_folder, self.pipeline_cache_list_file_name), mode='a') as file_:
-            next_cached_pipeline_path = self._create_next_cached_pipeline_path(pipeline_cache_folder)
-
-            file_.write(str(data_container))
-            file_.write('\n')
-            file_.write(next_cached_pipeline_path)
-            file_.write('\n')
-
-            dump(pipeline, next_cached_pipeline_path)
+        next_cached_pipeline_path = self._create_next_cached_pipeline_path(pipeline_cache_folder, data_container)
+        dump(pipeline, next_cached_pipeline_path)
```

Comment:
Cool, I get that more. 

And perhaps not importing dump directly and doing `joblib.dump` would avoid possible confusion (optional).